CONTRIBUTING.md

```diff
@@ -14,7 +14,9 @@ Open an issue and provide enough cotext information to repro the condition. Don'
 
 ### Environment Setup
 
-See the `BUILD PROCESS` section in the `README.md` file.
+See the `BUILD PROCESS` section in the `README.md` file. Alternatively if you have a web browser, you can get a fully pre-configured development environment in one click:
```

Comment:
Do you mind moving this to the `BUILD PROCESS` section in the main README file, so everything is in one place.

Reply:
Absolutely.